typescript/infra/package.json

```diff
@@ -14,9 +14,9 @@
     "@gnosis.pm/safe-core-sdk": "^2.3.2",
     "@gnosis.pm/safe-ethers-lib": "^1.4.0",
     "@gnosis.pm/safe-service-client": "^1.2.0",
-    "@hyperlane-xyz/helloworld": "1.3.7",
-    "@hyperlane-xyz/sdk": "1.3.7",
-    "@hyperlane-xyz/utils": "1.3.7",
+    "@hyperlane-xyz/helloworld": "1.3.8",
```

Comment:
Please increment infra package version above (yarn command doesn't do it cause this is a private pkg)